src/Services/FHIR/FhirPatientService.php

```diff
@@ -2,167 +2,382 @@
 
 namespace OpenEMR\Services\FHIR;
 
+use Ramsey\Uuid\Uuid;
```

Comment:
Not seeing where this class is being used in this script

Reply:
Good catch! I can remove the import/use statement.